src/binwalk/magic/firmware

```diff
@@ -468,11 +468,13 @@
 # Found in DIR-100 firmware
 0       string      AIH0        AIH0 firmware header, header size: 48,
 >12     ubelong     0           {invalid}
->12     ubelong+48  x           {size:%d}
+>12     ubelong     >0x4000000  {invalid}
+>12     ubelong     x           {size:%d}
```

Comment:
The size tag should usually include the header size (+48), since this is used internally by binwalk to know how far the data extends from the start of the signature.